traffic_portal/test/integration/PageObjects/UsersPage.po.ts

```diff
@@ -46,13 +68,39 @@ export class UsersPage extends BasePage {
     private txtPassword = element(by.name('uPass'));
     private txtConfirmPassword = element(by.name('confirmPassword'));
     private txtPublicSSHKey = element(by.name('publicSshKey'));
+    private txtSearch = element(by.id('usersTable_filter')).element(by.css('label input'));
+    private btnTableColumn = element(by.css('[title="Select Table Columns"]'));
     private randomize = randomize;
 
-    async OpenUserPage(){
-      let snp = new SideNavigationPage();
-      await snp.ClickUserAdminMenu();
-      await snp.NavigateToUsersPage();
-     }
+    async OpenUserPage() {
```

Comment:
Missing return type on call signature

Reply:
Why would I need a return type when the function isn't returning anything?

Reply:
That's why, because it actually does return something, and that's not immediately obvious:
```javascript
// A Promise that just resolves to 5
const p = new Promise(resolve=>resolve(5));
async function sideNavMethod() {
	await p; // discarding return value; just wanted to 'await' something
}

async function userPageMethod() {
	await sideNavMethod();
}

// outputs: object
console.log(typeof(userPageMethod()));
```